core/src/test/java/org/apache/calcite/test/JdbcTest.java

```diff
@@ -3803,6 +3803,42 @@ public void checkOrderBy(final boolean desc,
             "empid=110; name=Theodore");
   }
 
+  @Test void testMinusToDistinct1() {
```

Comment:
Nit: different tests are covering for different aspects of the feature, if possible it's better to convey what's the specificity of the test in its name, instead of appending a counter. 

In this case, I'd say the focus is involving a subquery, so what about `testMinusToDistinctWithSubquery` or similar?

Reply:
It makes sense for me.